js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/viewIndex/CollectionIndexDetails.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  ModalCloseButton,
+  Stack,
+  Tag
+} from "@chakra-ui/react";
+import React from "react";
+import { useRouteMatch } from "react-router-dom";
+import { ControlledJSONEditor } from "../../../../components/jsonEditor/ControlledJSONEditor";
+import { Modal, ModalBody, ModalFooter } from "../../../../components/modal";
+import { useCollectionIndicesContext } from "../CollectionIndicesContext";
+import { TYPE_TO_LABEL_MAP } from "../CollectionIndicesHelpers";
+import { DeleteIndexModal } from "../DeleteIndexModal";
+import { IndexStats } from "./IndexStats";
+
+export const CollectionIndexDetails = () => {
+  const match = useRouteMatch<{
+    indexId: string;
+  }>("/cIndices/:collectionName/:indexId");
+  const { collectionIndices, readOnly } = useCollectionIndicesContext();
+  const { indexId } = match?.params || {};
+  // need to use winodw.location.hash here instead of useRouteMatch because
+  // useRouteMatch does not work with hash router
+  const fromUrl = window.location.hash.split("#cIndices/")[1];
+  const collectionNameFromUrl = fromUrl.split("/")[0];
+  const decodedCollectionNameFromUrl = decodeURIComponent(
+    collectionNameFromUrl
+  );
+  const [showDeleteModal, setShowDeleteModal] = React.useState(false);
+  const currentIndex = collectionIndices?.find(collectionIndex => {
+    return collectionIndex.id === `${decodedCollectionNameFromUrl}/${indexId}`;
+  });
+  if (!currentIndex) {
```

Comment:
What if `currentIndex` is 0? Expected `null`?

Reply:
how can it be `0`? it's an object

Reply:
Yeah! Maybe we should 🙂